src/main/webapp/app/exercises/modeling/shared/modeling-editor.component.ts

```diff
@@ -24,7 +24,7 @@ export class ModelingEditorComponent implements AfterViewInit, OnDestroy, OnChan
     @Input()
     readOnly = false;
     @Input()
-    resizeOptions: { initialWidth: string; maxWidth?: number };
+    resizeOptions: { initialWidth?: string; maxWidth?: number; verticalResize?: boolean };
```

Comment:
Is there a reason why we don't have consistent flags for both vertical and horizontal resize?